packages/blockchain-wallet-v4-frontend/webpackBuilder.js

```diff
@@ -237,6 +237,7 @@ const buildDevServerConfig = (
           envConfig.WALLET_HELPER_DOMAIN,
           envConfig.WEB_SOCKET_URL,
           envConfig.OPENSEA_API,
+          'http://localhost:4444',
```

Comment:
whats this for? if needed can you add a comment explaining?

Reply:
Ah yeah good find we can delete now, was used for testing, thanks!